### ISSUE TITLE
Connect frontend to backend API

### ISSUE THREAD

**Issue by pkspyder007** [girlscript/feminist-bible-phase-2#157]
This is a discussion issue where you can suggest your ideas on how we might integrate our backend with the frontend.
e.g. Using Axios or react-query for data fetching, how to model global state, etc.

Separate issues will be created for specific functionality later on.

**Comment by gauravdas014** [comment]
According to me, Axios should be used for the API calls. Regarding the react part @salil-naik can comment better.

**Comment by rajesh6161** [comment]
I would like to handle this issue

**Comment by gauravdas014** [comment]
@rajesh6161 As you can see, it's kind of a discussion issue. There will be separate issues for connecting different components of front end to the backend.

**Comment by nishant23122000** [comment]
Hey as per me, we have many option to connect frontend with backend ,
 1> we can use passport or any other library for handling user cookies,sessions and all with easy methods .

2> we can implement alll things manually means handle cookie and sessions manually 
And axios is best because it provides many options like with-credentials etc and also there is big community , so that we can easily find the any query solution

Also i implemented all things in my project and it's works great😎

**Comment by gauravdas014** [comment]
@nishant23122000 Would you like to work on some part of the integeation? As the integration of the whole API will be a very big task. Please note we aren't using passport js. And please go through the API and also the react app for a better understanding of the codebase so that you can integrate the API properly.

**Comment by nishant23122000** [comment]
@gauravdas014  Yes, I like to contribute in this part because i already familiar on this and also i already understood the codebase of front and back

**Comment by gauravdas014** [comment]
@nishant23122000 Please create separate issues for each section of integration and start working
